modules/cis-alarms/main.tf

```diff
@@ -79,7 +79,7 @@ locals {
   ###############
 
   prefix   = var.use_random_name_prefix ? "${random_pet.this[0].id}-" : var.name_prefix
-  controls = { for k, v in local.all_controls : k => v if !contains(var.disabled_controls, k) }
+  controls = { for k, v in local.all_controls : k => merge(v, lookup(var.control_overrides, k, {})) if var.create && !contains(var.disabled_controls, k) }
```

Comment:
Thanks for making the change! I initially used `try`, but decided to stick with `lookup` because 1) wasn't sure in which version of terraform `try` was introduced and 2) the rest of the module already uses `lookup` for other variables

Reply:
👍  We use Terraform 0.13+ in almost all `terraform-aws-modules` where `try` was introduced, and we rewrite modules whenever we make some changes.